fabric-tool-attributes-v1/src/main/java/net/fabricmc/fabric/api/tools/v1/ToolActor.java

```diff
@@ -0,0 +1,17 @@
+package net.fabricmc.fabric.api.tools.v1;
+
+import com.sun.istack.internal.Nullable;
```

Comment:
:thinking:

Reply:
I *swore* that I imported javax Nullable.